### PR TITLE
add css lint

### PR DESCRIPTION
**Pull request by AnyaTyu**

### CODE REVIEW

**Comment by tadatuta** [comment]
lgtm

File: gulpfile.js [hunk 1]
```diff
@@ -201,7 +201,7 @@ gulp.task('libs-build', function() {
     // require('bem-lib-site-generator').data(path.resolve('path/to/library'));
 });
 
-gulp.task('styles', function() {
+gulp.task('ccscomb', function() {
```
Comment (vithar) [inline gulpfile.js]: ccs -> css

File: gulpfile.js [hunk 2]
```diff
@@ -215,7 +215,7 @@ gulp.task('default', (done) => runSequence(
     'copy-static-images',
     'copy-sitemap-xml',
     'build-html',
-    'styles',
+    'ccscomb',
```
Comment (vithar) [inline gulpfile.js]: ccs -> css

Comment (AnyaTyu) [inline gulpfile.js]: :satisfied: исправила

**Pull request merged**